### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ivanebernal**
Added cardview, adapter and fields to the ToDoContent class.

### CODE REVIEW

File: todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/ToDoContent.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package com.nearsoft.androidschool.todoapp;
+
+
+/**
+ * Created by Ramon on 2/1/2017.
+ */
+public class ToDoContent/* extends Activity */{
```
Comment (RoyMontoya) [inline todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/ToDoContent.java:7]: remove the comment "extends..." this class looks like a model

File: todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/ToDoListAdapter.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    public ToDoListAdapter(Context context, List<ToDoContent> contents) {
+        mContext = context;
+        mContents = contents;
+    }
+
+    @Override
+    public ToDoListAdapter.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        LayoutInflater inflater = LayoutInflater.from(mContext);
+        View item = inflater.inflate(R.layout.list_item, parent, false);
```
Comment (RoyMontoya) [inline todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/ToDoListAdapter.java:28]: change the name to either "itemView" or "view" the method expects a View type.

File: todoapp/TODOapp/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">TODOapp</string>
+    <string name="hello_world">Hello world</string>
```
Comment (RoyMontoya) [inline todoapp/TODOapp/app/src/main/res/values/strings.xml]: we are not gonna need this string, delete it

File: todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/ToDoContent.java (hunk 1 above)
Comment (jmsalcido) [reply todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/ToDoContent.java:7]: addressed on 2a0f7b90a7a8343b4abb07d5251c28f8aa5c1759

File: todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/ToDoListAdapter.java (hunk 2 above)
Comment (jmsalcido) [reply todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/ToDoListAdapter.java:28]: addressed on 2a0f7b90a7a8343b4abb07d5251c28f8aa5c1759

**Pull request merged**